umap/fields.py

```diff
@@ -14,7 +15,7 @@ def get_prep_value(self, value):
         if not value:
             value = {}
         if not isinstance(value, six.string_types):
-            value = json.dumps(value)
+            value = json.dumps(value, cls=DjangoJSONEncoder)
```

Comment:
Ah, this field may only be still there for the migration process. Maybe we could move it in the migration file iself, but that's another story of what this PR is about.

Reply:
> One question: I wonder why we needed this, as I don't a good reason to store UUIDs in those fields…

I iterated over all the `json.dumps` to use the Django JSON Encoder, for consistency. It might not be useful in this specific case.